arches_lingo/src/arches_lingo/components/scheme/SchemeStandard/SchemeStandard.vue

```diff
@@ -38,8 +40,6 @@ onMounted(async () => {
         const sectionValue = await getSectionValue();
         tileData.value = sectionValue[props.nodegroupAlias];
     }
-
-    isLoading.value = false;
```

Comment:
because `getSectionValue` is within an if-statement, removing `isLoading.value = false` from here can result in the component never leaving the loading state.